tests/wicked/2nics/ref/t03_setup_second_card.pm

```diff
@@ -16,13 +16,18 @@ use base 'wickedbase';
 use strict;
 use warnings;
 use testapi;
+use lockapi;
+use utils 'systemctl';
 
 
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
     my $ip_address = $self->get_ip(type => 'second_card');
     assert_script_run("ip a a $ip_address dev " . $ctx->iface());
+    $self->get_from_data('wicked/dhcp/dhcpd_2nic.conf', '/etc/dhcpd.conf');
+    systemctl('restart dhcpd.service');
+    die("Create mutex failed") unless mutex_create('t03_dhcpd_setup_complete');
```

Comment:
why you think we need die in this case ? all mutexes whuch we creating in `lock_init` don't have any dies

Reply:
I just realized that this function only return false on failure. But it could happen.